src/argilla/server/contexts/datasets.py

```diff
@@ -1092,20 +1092,32 @@ async def upsert_suggestion(
 
 async def delete_suggestions(db: "AsyncSession", record: Record, suggestions_ids: List[UUID]) -> None:
```

Comment:
this code may not be affected?

Reply:
It's clearly affected yes. I will take a look.